test/data/jquery-1.9.1.js

```diff
@@ -2727,7 +2727,7 @@ if ( !jQuery.support.style ) {
 
 			// Return undefined in the case of empty string
 			// Note: IE uppercases css property names, but if we were to .toLowerCase()
-			// .cssText, that would destroy case senstitivity in URL's, like in "background"
+			// .cssText, that would destroy case sensitivity in URL's, like in "background"
```

Comment:
this file is a fixed library, even if it's not a change to the api, it should remain as it is.